examples/webgpu_materials.html

```diff
@@ -370,7 +370,7 @@
 
 				//
 
-				// setTimeout( () => testSerialization( serializeMesh ), 1000 ); // FIXME, see #28933
+				setTimeout( () => testSerialization( serializeMesh ), 1000 ); // FIXME, see #28933
```

Comment:
@Mugen87 Do you need to remove the inline comment?

Reply:
28636b09ca1d0c4bb31dc34ca2d3f73d12865352

Reply:
It is rendering a black teapot on a black background. It does not appear to be working as intended...

Reply:
Yes, there is an issue with the material but this should be unrelated to serialization/deserialization. Probably best to file a separate issue for this.